docs/whats-new/zowe-v3-frequently-asked-questions.md

```diff
@@ -0,0 +1,123 @@
+
+5. Is it possible to validate and change the `zowe.yaml` file and job definition within ZEN?
+
+Yes.
+
+6. What happens if in ZEN you click on the SMP/E? **[what is smp/e here? is it a button? a drop-down option? how do users get to smp/e?]**
+
+You are guided through the SMP/E installation steps.
+
+7. What is the connection between Zowe Store and ZEN?
+
+Zowe Store is separate from ZEN, but they do share **[some commands? all commands?]** `zwe` commands.
+
+8. Does the Zowe Store handle Zowe upgrades?
+
+No, it offers server side extensions.
+
+### Zowe System Installation and Configuration
+
+1. Does Zowe provide a Software Bill of Materials (SBOM)?
+
+   Zowe V3 provides SBOMs in an SPDX format. **[have there been changes to the format? if so, what are the changes?]**
+
+## User questions
+
+### Zowe API Mediation Layer
+
+1. Will Zowe V3 APIML support clients with onboarding enabler V2?
+
+   The API ML in Zowe V3 supports clients with onboarding enablers from Zowe V2. It also works the other way around: The onboarding enablers from Zowe V3 will continue working with Zowe V2 API ML.
+
+2. The jump to Java 17 means that we have to maintain two separate versions of our application, one with apiml-enabler V2 and one with V3, to support customers who want to stay with Java 8. Do you have a recommendation or a workaround for supporting both Java versions?
+
+   The plain Java Enabler from Zowe V2 works in Zowe V3. You can keep this until Zowe V4.
+
+   Zowe V4 will only support Spring Boot, which requires Java 17. We recommend building applications for Java 17 to ensure that applications continue to be compatible with API ML.
+
+3. Can you speak about the migration from Zuul to Spring Cloud Gateway? Today there are two separate gateway services in API ML with separate config.
+
+   We expect to support only Spring Cloud Gateway. The spring-cloud-gateway configuration should move under the component.gateway namespace. We expect to have **[should users be doing the config here?]** one Spring Cloud Gateway cluster ahead **[ahead or between?]** of multiple sysplexes and then one cluster on every sysplex. Most of the configuration that was used should remain, including the new one for Spring Cloud Gateway.
+
+4. If I have a legacy gateway deployed, how will I migrate to the new gateway? Will the old gateway be removed?
+
+   The old gateway is removed in Zowe V3, but nothing should change from the point of view of the user. However, we may deliver a configuration utility to help with this transition, if needed.
+
+5. How would a client-side end user find and obtain the correct API ML service instance ID for the desired instance of their service?
+
+   In Zowe V2 **[is V2 correct here?]**, clients can use the header instance ID to route communications to a specific instance. Clients can get instance IDs for specific services via an API on the discovery service. We are planning to improve the method for finding service IDs in Zowe V3.
```

Comment:
@balhar-jakub - please respond

Reply:
Yes, it's supported in V2 and will continue to be supported in V3 with improvements to the method.